### DISCUSSION TITLE
[Accessibility-metadata] What machine readability means here?

### DISCUSSION THREAD

**Discussion by abhina-kumar** [w3c/personalization-semantics#257] [Ideas]
1) Accessibility-metadata is meant to be machine readable also. What is machine readability means in this context?

2) Further, it is mentioned - `machine readability may give some organisations concerns (as it makes researching easier).` Most of the content in Accessibility-metadata is already public....Can we have details on what are the possible concerns?

**Comment by lwolberg** [comment]
Discussed in the Task Force today.

**Comment by lwolberg** [comment]
We are prioritizing human readable in the first stage of planning Well Known Destination.
